server/src/security/authorisationService.ts

```diff
@@ -172,6 +178,39 @@ export async function getResources<S extends WithSecurityScheme>(schema: S, req:
   }
 }
 
+const getResourceIds = (resources: Resources): ResourceIds => {
+  const resourcesIds: ResourceIds = {}
+  if (resources.recruiters.length) {
+    resourcesIds.recruiters = resources.recruiters.map((recruiter) => recruiter._id.toString())
+  }
+  if (resources.users.length) {
+    resourcesIds.users = resources.users.map((user) => user._id.toString())
+  }
+  if (resources.jobs.length) {
+    resourcesIds.jobs = resources.jobs.map((job) =>
+      job
+        ? {
+            job: job.job._id.toString(),
+            recruiter: job.recruiter ? job?.recruiter._id.toString() : null,
+          }
+        : null
+    )
+  }
+  if (resources.applications.length) {
+    resourcesIds.applications = resources.applications.map((application) =>
+      application
+        ? {
+            application: application.application._id.toString(),
+            job: application.job._id.toString(),
+            recruiter: application.recruiter._id.toString(),
+          }
+        : null
+    )
+  }
```

Comment:
Je penses qu'il faut changer la manière de faire ici, le jour ou on ajoute un nouveau type de resource il sera certainement oublié dans ce bout de code. On peut refactor comme ça histoire d'avoir Typescript et l'APP qui plante lors de l'ajout d'un nouveau type de ressource.

```suggestion
Object.keys(resources).map((key) => {
   switch (key) {
      case 'recruiters':
         ...
      case ...:
      default:
         assertUnreachable(key);
   }
});
```

cc: @remy-auricoste

Reply:
Done ! (si tu peux recheck ce bloc stp)